src/main/scala/tech/sourced/gemini/QueryApp.scala

```diff
@@ -46,7 +55,8 @@ object QueryApp extends App {
       val gemini = Gemini(null, log)
       gemini.applySchema(cassandra)
 
-      val similar = gemini.query(file, cassandra)
+      val client = BblfshClient.apply(config.bblfshHost, config.bblfshPort)
+      val similar = gemini.query(file, cassandra, client)
```

Comment:
Just a suggestion - up to you, but merely for the sake of consistency, as Cassandra client var is called `cassandra`, it may be worth calling bblfsh client `bblfsh` here.

Reply:
Seems like this might have been addressed in https://github.com/src-d/gemini/pull/83/commits/6e82586ce94060e7d8e13c5d7110498d9ee26656#diff-768357618a8db6a4da444273c800e8d9R68

Reply:
yep. Absolutely agree and addressed in next pr.